llvm/lib/Target/AVR/CMakeLists.txt

```diff
@@ -7,7 +7,7 @@ tablegen(LLVM AVRGenAsmWriter.inc -gen-asm-writer)
 tablegen(LLVM AVRGenCallingConv.inc -gen-callingconv)
 tablegen(LLVM AVRGenDAGISel.inc -gen-dag-isel)
 tablegen(LLVM AVRGenDisassemblerTables.inc -gen-disassembler)
-tablegen(LLVM AVRGenInstrInfo.inc -gen-instr-info)
+tablegen(LLVM AVRGenInstrInfo.inc -gen-instr-info --emit-operand-types)
```

Comment:
(nit) double dash is inconsistent with other changes

Reply:
Fixed. I guess cl:: library accepts both, so things worked (I built locally with all backends enabled).